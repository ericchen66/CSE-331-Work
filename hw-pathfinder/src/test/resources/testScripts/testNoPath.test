# Tests Dijkstra's algorithm when there is no
# path between two nodes

CreateGraph simpleGraph

AddNode simpleGraph A
AddNode simpleGraph B
AddNode simpleGraph C

AddEdge simpleGraph A B 1.0

FindPath simpleGraph A C