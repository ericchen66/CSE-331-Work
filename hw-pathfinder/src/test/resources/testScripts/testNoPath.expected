# Tests Dijkstra's algorithm when there is no
# path between two nodes

created graph simpleGraph

added node A to simpleGraph
added node B to simpleGraph
added node C to simpleGraph

added edge 1.000 from A to B in simpleGraph

path from A to C:
no path found